packages/cozy-harvest-lib/src/components/ModalBackButton.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import cx from 'classnames'
+import flow from 'lodash/flow'
+import Button from 'cozy-ui/transpiled/react/Button'
+import withBreakpoints from 'cozy-ui/transpiled/react/helpers/withBreakpoints'
+import withLocales from './hoc/withLocales'
+
+const DumbModalBackButton = props => {
```

Comment:
On n'a rien dans UI pour ça ? Si on n'y a rien, je pense vraiment que c'est là bas qu'il faut le mettre (et si on a quelque chose, qu'il faut réutiliser)

Reply:
Extracted it to cozy-ui. See https://github.com/cozy/cozy-ui/pull/1184

Reply:
👍